gslib/commands/ls.py

```diff
@@ -416,6 +418,7 @@ def _PrintBucketInfo(self, bucket_blr, listing_style):
     text_util.print_to_fd(
         ('{bucket} :\n'
          '\tStorage class:\t\t\t{storage_class}\n'
+         '\tLocation type:\t\t\t{location_type}\n'
```

Comment:
Is this field going to be present in XML API responses too? If not, let's not bother printing `Location type: None` if the XML API was used (we do the same thing for other JSON-only fields - see `time_updated_line` and other `<foo>_line` variables).

Reply:
Good call. I'm fixing that now.